client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -1,76 +1,60 @@
-                        <div class="col-md-8">
-                            <span class="d-block form-control-static"
-                                  data-ng-if="!configurationsCtrl.edit.value">
-                                <span data-ng-bind="configuration.value"></span>
-                                <span data-i18n-static="{{'unit_configuration_' + configuration.unit}}" data-ng-if="configuration.unit"></span>
-                            </span>
-                            <div class="oui-select oui-select_inline" data-ng-if="configurationsCtrl.edit.value">
-                                <select class="oui-select__input" name="{{configuration.key}}"
-                                        data-ng-attr-id="{{configuration.key + 'Field'}}"
-                                        data-ng-model="configuration.value"
-                                        data-ng-options="conf for conf in configuration.availableValues">
-                                </select>
-                                <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
-                            </div>
-                        </div>
-                    </div>
-                </form>
-            </div>
-        </div>
+            <h2 data-ng-bind-html="tr('privateDatabase_configurations_general_' + configurationsCtrl.database.type)"></h2>
 
-        <div class="col-md-3 mt-5 mt-lg-0"
-             data-ng-if="configurationsCtrl.database.type !== 'postgresql'">
-            <button class="btn btn-block btn-default" type="button"
-                    data-i18n-static="privateDatabase_configuration_edit_btn"
-                    data-ng-click="configurationsCtrl.edit.value = true"
-                    data-ng-disabled="database.expired || isRebooting"
-                    data-ng-if="!configurationsCtrl.edit.value">
-            </button>
-            <button class="btn btn-block btn-primary" type="button"
-                    data-i18n-static="privateDatabase_configuration_confirm_btn"
-                    data-ng-click="configurationsCtrl.updateConfigurations()"
-                    data-ng-disabled="database.expired"
-                    data-ng-if="configurationsCtrl.edit.value">
-            </button>
-            <button class="btn btn-block btn-default" type="button"
-                    data-i18n-static="privateDatabase_configuration_cancel_btn"
-                    data-ng-click="configurationsCtrl.configurations = configurationsCtrl.baseConfiguration; configurationsCtrl.edit.value = false"
-                    data-ng-disabled="database.expired"
-                    data-ng-if="configurationsCtrl.edit.value">
-            </button>
+            <form name="dbConfigurationForm" data-ng-submit="configurationsCtrl.updateConfigurations()">
+                <div data-ng-repeat="configuration in configurationsCtrl.configurations track by $index">
+                    <oui-field label="{{configuration.key}}"
```

Comment:
`data-`

Reply:
@FredericEspiau http://master.ui-kit.ovh/#!/oui-angular/decisions ;)

Reply:
Ui-kit decisions are only for ui-kit repos.. for now, we keep data- for all managers

Reply:
ok @tremblaymath. thx.